blueprints/addon/files/README.md

```diff
@@ -1,26 +1,47 @@
-# <%= addonName %>
 
-This README outlines the details of collaborating on this Ember addon.
+<%= addonName %>
+==============================================================================
 
-## Installation
+[Short description of the addon.]
 
-* `git clone <repository-url>` this repository
+Installation
+------------------------------------------------------------------------------
+
+```
+ember install <%= addonName %>
+```
+
+
+Usage
+------------------------------------------------------------------------------
+
+[Longer description of how to use the addon in apps.]
+
+
+Contributing
+------------------------------------------------------------------------------
+
+### Installation
+
+* `git clone <repository-url>`
 * `cd <%= addonName %>`
 * `<% if (yarn) { %>yarn<% } else { %>npm<% } %> install`
 
-## Running
+### Running tests
 
-* `ember serve`
-* Visit your app at [http://localhost:4200](http://localhost:4200).
+* `ember test` – Runs the test suite on the current Ember version
+* `ember test --server` – Runs the test suite in "watch mode"
+* `<% if (yarn) { %>yarn<% } else { %>npm<% } %> test` – Runs `ember try:each` to test your addon against multiple Ember versions
+
+### Running the dummy application
 
-## Running Tests
+* `ember serve`
+* Visit the dummy application at [http://localhost:4200](http://localhost:4200).
 
-* `<% if (yarn) { %>yarn<% } else { %>npm<% } %> test` (Runs `ember try:each` to test your addon against multiple Ember versions)
-* `ember test`
-* `ember test --server`
+For more information on using ember-cli, visit [https://ember-cli.com/](https://ember-cli.com/).
 
-## Building
 
-* `ember build`
+License
+------------------------------------------------------------------------------
 
-For more information on using ember-cli, visit [https://ember-cli.com/](https://ember-cli.com/).
+This project is licensed under the [MIT License](LICENSE.md).
```

Comment:
This will make a third location that the license of the addon is indicated (package.json & LICENSE.md being the others).

Not sure that I object, but feel it is worth noting.

Reply:
Yeah I wouldn't mind removing this section either.

Reply:
I would prefer to keep it in for better visibility 🤔